ci/lava/README.md

```diff
@@ -0,0 +1,17 @@
+# Test setup
+
+`ci/lava` directory contains directories with names corresponding to build MACHINE names.
+New directories should only be added after the build is available for a given MACHINE.
+All files in the `ci/lava/<MACHINE>` directory should have `.yaml` extension.
+Each file should be a valid LAVA job template.
```

Comment:
Is there a way to validate the job template?

Reply:
Easiest is to submit it to LAVA through UI. There is a validator script - part of LAVA sources, but it's not standalone and it's hard to use. There is also an API, so in theory I can validate the files as part of github wokflow:
https://lava.infra.foundries.io/api/v0.2/jobs/validate/
Do you think it's a good idea?

Reply:
My main concern here is mostly to document how to validate the files, don't need something fancy. Just extending here saying how you / someone can validate would be fine.

Reply:
There is. There is also a validator in LAVA code: https://gitlab.com/lava/lava/-/tree/master/lava_common/schemas?ref_type=heads. Templates can be validated using the running server, but it requires password in HTTP call. I guess the easiest way to use the existing LAVA implementation would be to call the lava container in github action. I'll try to propose additional check. We'll need to keep the version of the server in sync with the check though. It's a minor issue as schema rarely changes these days.

Reply:
I added a script. This should trigger an action checking the templates.